include/pistache/http.h

```diff
@@ -472,6 +479,20 @@ namespace Pistache
                 return nullptr;
             }
 
+            // Compress using the requested content encoding, if supported,
+            //  before sending bits to client. User responsible for setting
+            //  Content-Encoding header...
```

Comment:
_"User responsible for setting Content-Encoding header"_ this is no longer the case

Reply:
Fixed.